e2e/Disabled/anvilTransactionFlowSendAndWC.disable.js

```
@@ -0,0 +1,397 @@
+          if (error) {
+            reject(error);
+          }
+          const { accounts } = payload.params[0];
+          if (accounts[0] === TESTING_WALLET) {
+            account = accounts[0];
+            resolve(true);
+          } else {
+            reject(false);
+          }
+        });
+      });
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)